pom.xml

```diff
@@ -445,12 +445,16 @@
             </goals>
             <configuration>
               <failOnWarning>true</failOnWarning>
+              <ignoredUsedUndeclaredDependencies>
+                <usedUndeclaredDependency>com.google.auto.service:auto-service-annotations:jar:*</usedUndeclaredDependency>
+              </ignoredUsedUndeclaredDependencies>
               <ignoredUnusedDeclaredDependencies>
                 <!-- auto-service isn't detected as use since the annotation has retention of source -->
                 <unusedDeclaredDependency>com.google.auto.service:auto-service:jar:*</unusedDeclaredDependency>
                 <!-- unused/declared implementation jars or parent jars that bring in children -->
                 <unusedDeclaredDependency>org.apache.hadoop:hadoop-client-runtime:jar:*</unusedDeclaredDependency>
                 <unusedDeclaredDependency>org.slf4j:slf4j-log4j12:jar:*</unusedDeclaredDependency>
+                <unusedDeclaredDependency>org.slf4j:slf4j-reload4j:jar:*</unusedDeclaredDependency>
```

Comment:
If this is being added, the log4j12 should be removed.

Reply:
Fixed in [21abb1a](https://github.com/apache/accumulo-proxy/pull/55/commits/21abb1a407f0ea857f7f6bfa3bb636a8c282b51c)